src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/healthChart.tsx

```diff
@@ -69,12 +70,23 @@ class HealthChart extends React.Component<Props> {
     }
   };
 
+  getChart = () => {
```

Comment:
maybe you could return the type here or pass the props to this method and return the components like <AreaChart {...props} />